client/src/components/MapSection.css

```diff
@@ -21,10 +19,45 @@
 	z-index: 2;
 }
 
-.help-text {
+.night {
+	background-image: linear-gradient(
+		to right,
+		#0c1330,
+		#0c1330,
+		#182049,
+		#1e2c5f,
+		#182049,
+		#0c1330,
+		#0c1330
+	);
+
+	color: antiquewhite;
+}
+
+.bootstrap-container {
 	margin-bottom: 70px;
 	padding-left: 82px;
 	padding-top: 20px;
+	display: flex;
+	justify-content: space-evenly;
+	align-items: flex-start;
+}
+
+.bootstrap-container span {
+	margin-top: 10px;
+}
+
+.bootstrap-light {
+	background-color: white;
+	box-shadow: 10px 10px 12px -8px rgba(0, 0, 0, 0.75);
+	color: #cb3100;
```

Comment:
Add a border or different background color in light mode to distinguish that it's a button